components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/apis/{apiId}/index.js

```diff
@@ -211,6 +211,152 @@ $(function () {
+	                                                            	for (var j = 0; j < keys.length; j++) {
+	                                                            		if (keys[j].keyType == "PRODUCTION") {
+	                                                            			app["prodKey"] = keys[j].token.accessToken;
+	                                                            		} else {
+	                                                            			app["sandBKey"] = keys[j].token.accessToken;
+	                                                            		}
+	                                                            	}
+	                                                            	subscribedAppsAndKeys.push(app);
+	                                                            	context.subscribedAppsAndKeys = subscribedAppsAndKeys;
+	    	                                                        
+	    	                                                        UUFClient.renderFragment("org.wso2.carbon.apimgt.web.store.feature.api-console-auth",context, {
+	    	                                                			onSuccess: function (renderedData) {
+	    	                                                				$("#authorizations").html(renderedData);
+	    	                                                				setOverviewTabData();
+	    	                                                			}.bind(context), onFailure: function (message, e) {
+	    	                                                				var message = "Error occurred while loading API console." + message;
+	    	                                                                noty({
+	    	                                                                    text: message,
+	    	                                                                    type: 'error',
+	    	                                                                    dismissQueue: true,
+	    	                                                                    modal: true,
+	    	                                                                    progressBar: true,
+	    	                                                                    timeout: 2000,
+	    	                                                                    layout: 'top',
+	    	                                                                    theme: 'relax',
+	    	                                                                    maxVisible: 10,
+	    	                                                                });
+	    	                                                			}
+	    	                                                		});
+	                                                            },
+	                                                            function (error) {
+	                                                                alert("Error occurred while retrieving Application details" + error);
```

Comment:
Shall we use noty to display errors instead of using method "alert". IMHO it would be nice to have.